src/components/signed-out/signed-out-routes.ts

```diff
@@ -0,0 +1,10 @@
+import { PATH_DATA } from "../../app.constants";
+
+import * as express from "express";
```

Comment:
possible to avoid the wildcard import?

Reply:
Sure - changed to named import of Router, as only this is used